src/viewport/index.tsx

```diff
@@ -402,6 +420,9 @@ class Viewport extends React.Component<
                 this.visGeometry.destroyGui();
             }
         }
+        if (playbackSpeed !== prevProps.playbackSpeed) {
+            this.clockTimePerFrameAdvance = 1000 / playbackSpeed;
```

Comment:
Do you need to store `clockTImePerFrameAdvance` or can we just calculate it off of `props.playbackSpeed` each time?